### PR TITLE
frontend: Node: Fix table columns and empty values

### PR DESCRIPTION
**Pull request by skoeva**
This change adjusts the width of the table columns to prevent UI issues and fills in empty values where applicable.

Fixes: #2100

### Testing
- [X] Open Headlamp in cluster and navigate to the Nodes page
- [X] Ensure that all empty fields are filled with "None" and columns with the info icon have sufficient spacing.

image

### CODE REVIEW

File: frontend/src/components/node/List.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default function NodeList() {
         {
           id: 'cpu',
           label: t('CPU'),
+          gridTemplate: 'minmax(120px, .5fr)',
```
Comment (joaquimrocha) [inline frontend/src/components/node/List.tsx:29]: Why is this needed?

File: frontend/src/components/node/List.tsx [hunk 2]
```diff
@@ -58,7 +59,7 @@ export default function NodeList() {
         {
           id: 'ready',
           label: t('translation|Ready'),
-          gridTemplate: 'minmax(150px, .3fr)',
+          gridTemplate: 'minmax(120px, .3fr)',
```
Comment (joaquimrocha) [inline frontend/src/components/node/List.tsx:62]: maybe we could just delete this line?

File: frontend/src/components/node/List.tsx (hunk 1 above)
Comment (skoeva) [reply frontend/src/components/node/List.tsx:29]: Otherwise we get this in the UI (described in the issue):

image

File: frontend/src/components/node/List.tsx (hunk 2 above)
Comment (skoeva) [reply frontend/src/components/node/List.tsx]: your suggestion actually looks cleaner, removed the line here

File: frontend/src/components/node/List.tsx (hunk 1 above)
Comment (joaquimrocha) [reply frontend/src/components/node/List.tsx:29]: Maybe try `gridTremplate: 'min-content'`?

Comment (skoeva) [reply frontend/src/components/node/List.tsx]: seems to look very similar, made the change here for simplicity

**Review by joaquimrocha — APPROVED** [review]
